lib/src/endpoints/categories.dart

```diff
@@ -1,11 +1,11 @@
-part of spotify;
+part of '../../spotify.dart';
```

Comment:
Is this a new nomenclature I missed?

Also is using `part of '...'` still state of the art? I think it was weird to use 5 years ago and now it looks even weirder imo.

Reply:
This is the recommended way to pass to the last lints. `lints 3.0.0` is the current version recommended for Dart 3.3.0 (used by `dart create`)